backend/api/v1/debug_interceptor.go

```diff
@@ -31,30 +32,46 @@ func NewDebugInterceptor(errorRecordRing *api.ErrorRecordRing) *DebugInterceptor
 
 // DebugInterceptor is the unary interceptor for gRPC API.
 func (in *DebugInterceptor) DebugInterceptor(ctx context.Context, request any, serverInfo *grpc.UnaryServerInfo, handler grpc.UnaryHandler) (any, error) {
+	startTime := time.Now()
 	resp, err := handler(ctx, request)
 	if err != nil {
-		in.debugInterceptorDo(ctx, request, serverInfo.FullMethod, err)
+		in.debugInterceptorDo(ctx, request, serverInfo.FullMethod, err, startTime)
 	}
 
 	return resp, err
 }
 
 // DebugStreamInterceptor is the unary interceptor for gRPC API.
 func (in *DebugInterceptor) DebugStreamInterceptor(request any, ss grpc.ServerStream, serverInfo *grpc.StreamServerInfo, handler grpc.StreamHandler) error {
+	startTime := time.Now()
 	err := handler(request, ss)
 	ctx := ss.Context()
 	if err != nil {
```

Comment:
For OK, is this err == nil?

Reply:
For err == nil, status.Convert(err) get codes.OK

Reply:
In this case, shall we remove the error check wrapper?

`
	if err != nil {
  }
`

Reply:
If we do this, I think it will become another logging middleware.

Reply:
That's alright. We have a simpler one to use.

Reply:
Then we also need to modify log.BBError(), because it cannot handle `nil` correctly;

Reply:
Yeah, either that way or just do it similar to what logLevel, logMsg's are set.